models/speciality_list_view.py

```diff
@@ -7,4 +7,5 @@
 speciality_list_view = Table('speciality_list_view', metadata_obj,
           Column('university_id', Integer),
           Column('speciality_id', Integer),
-          Column('speciality_info', JSON))
+          Column('faculty_id', Integer),
```

Comment:
please make sure that order of sqlachemy columna are the same that in sql view
for ex: in view: faculty_id col - 1 st place, but there is this column on 3rd place